buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring.groovy

```diff
@@ -69,40 +73,69 @@ class GrpcVendoring {
     // ensures that there are no classes outside of the 'org.apache.beam' namespace.
 
     String prefix = "org.apache.beam.vendor.grpc.v1_13_1";
-    return {
+    return [
       // guava uses the com.google.common and com.google.thirdparty package namespaces
-      relocate "com.google.common", "${prefix}.com.google.common"
-      relocate "com.google.thirdparty", "${prefix}.com.google.thirdparty"
+      "com.google.common": "${prefix}.com.google.common",
+      "com.google.thirdparty": "${prefix}.com.google.thirdparty",
 
-      relocate "com.google.protobuf", "${prefix}.com.google.protobuf"
-      relocate "com.google.gson", "${prefix}.com.google.gson"
-      relocate "io.grpc", "${prefix}.io.grpc"
-      relocate "com.google.auth", "${prefix}.com.google.auth"
-      relocate "com.google.api", "${prefix}.com.google.api"
-      relocate "com.google.cloud", "${prefix}.com.google.cloud"
-      relocate "com.google.logging", "${prefix}.com.google.logging"
-      relocate "com.google.longrunning", "${prefix}.com.google.longrunning"
-      relocate "com.google.rpc", "${prefix}.com.google.rpc"
-      relocate "com.google.type", "${prefix}.com.google.type"
-      relocate "io.opencensus", "${prefix}.io.opencensus"
+      "com.google.protobuf": "${prefix}.com.google.protobuf",
```

Comment:
nit: these all seem to follow the same convention: `"original.package": "${prefix}.original.package",` It would be slightly less verbose to maintain just a list of `original.package`'s and generate the map from it.

Reply:
Done, a little harder to read whats going on now though.